### ISSUE TITLE
Findings for VSA spot check review for Original Claims

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#8512]
## Team Name

Benefits and Memorials 1

## Product/Feature Name

Original Claims

## Product description

Update to 526 form to allow for Original Claims to be submitted on VA.gov.  No front-end changes but review of 526 can be captured here as well since it's likely been a while since an accessibility review was conducted.

## Review stage

- [x] **Spot check review** (Review and follow the steps in the [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md) first.) This happens well before launch, so that the 508 office can review and any issues addressed *before* launch.

## Point of contact

Luke Majewski

## Preferred times for dev intent checkpoint

No preferred time other than open spot in my calendar (should have access to it)

## Staging URL and credentials

https://staging.va.gov/disability/how-to-file-claim/
Please contact me for staging account.

May 5, 2020: [Andrea Schneider said to use user 264](https://dsva.slack.com/archives/C5AGLBNRK/p1588706816199000?thread_ts=1588705836.197400&cid=C5AGLBNRK). 

## Test plan

Follow the normal 526 claims process since this was just back end changes.  Working on understanding whether we can do some TestRail for this.
